dts/bindings/display/nrf-led-matrix.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Generic LED matrix driven by nRF SoC GPIOs
+
+compatible: "nrf-led-matrix"
```

Comment:
should be `nrf,led-matrix`

Reply:
I used "nordic,nrf-led-matrix". The first part should be the vendor name, right?

Reply:
correct